charts/krill/templates/deployment.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "krill.fullname" . }}
+  labels:
+    {{- include "krill.labels" . | nindent 4 }}
+spec:
+{{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
+{{- end }}
+  selector:
+    matchLabels:
+      {{- include "krill.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+    {{- with .Values.podAnnotations }}
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      labels:
+        {{- include "krill.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "krill.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          ports:
+            - name: http
+              containerPort: 3000
+              protocol: TCP
+          volumeMounts:
+            - name: krill-conf
+              mountPath: {{ .Values.krillConf.data_dir }}/krill.conf
+              subPath: krill.conf
+            - name: data
+              mountPath: {{ .Values.krillConf.data_dir }}
+              subPath: {{ .Values.persistence.subPath | default "" | quote }}
+            {{- if and .Values.persistence.enabled .Values.persistence.rrdp.enabled }}
+            - name: rrdp-data
+              mountPath: {{ .Values.krillConf.data_dir }}/rrdp
```

Comment:
In the registry testbed, the krill instance publishes rrdp data under `./data/repo/rrdp`. Over here, the `rrdp-data` pv is mounted at `./data` instead.